### ISSUE TITLE
Skipped heading ranks

### ISSUE THREAD

**Issue by jcoyne** [LD4P/sinopia_editor#2477]
We should not skip heading ranks:

https://www.w3.org/WAI/tutorials/page-structure/headings/

But we do:
<img width="266" alt="Screen Shot 2020-09-14 at 1 23 37 PM" src="https://user-images.githubusercontent.com/92044/93123416-a8246300-f68d-11ea-9b3a-e39c4a4b268b.png">

**Comment by justinlittman** [comment]
@astridu Can you please determine if this should be fixed?

**Comment by astridu** [comment]
I think this is ok. It just gets confusing with the screen reader if you go from h2 to h4 bc the user might think it skipped a section.
